app/src/main/res/layout/main_activity.xml

```diff
@@ -59,6 +59,13 @@
         android:dividerHeight="1dp"
         android:verticalSpacing="4dp">
 
+        <EditText
+            android:id="@+id/UserMessage"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:ems="10"
+            android:hint="@string/song_name"
+            android:inputType="textPersonName" />
```

Comment:
What is this new UI element for?

Reply:
If it's not used anywhere, it should be removed.  Only features that have met the team's Definition of Done should be merged in master.